io/eolearn/io/local_io.py

```diff
@@ -10,10 +10,11 @@
 import dateutil
 import rasterio
 import numpy as np
+from pyproj import Proj, transform
+from sentinelhub import CRS, Bbox
```

Comment:
I made a typo in my merge commit via the GH UI, can you update this to `BBox` @AleksMat ? Sorry for the trouble

Reply:
Ok, no problem. It is fixed now.

Reply:
By the way, what was the reason for your last commit? Those imports (e.g. `from pyproj import Proj, transform`) were removed because they were not needed anymore.

Reply:
Probably another merge mistake 🤦‍♂ I'll remove them to my branch after merging this